sim/pkg/model/autoscaler_ticktock.go

```diff
@@ -62,6 +63,16 @@ func (asts *autoscalerTicktockStock) Add(entity simulator.Entity) error {
 	currentTime := asts.env.CurrentMovementTime()
 
 	asts.cluster.RecordToAutoscaler(&currentTime)
+
+	asts.adjustHorizontally(&currentTime)
+	asts.adjustVertically(&currentTime)
+
+	//calculate CPU utilization
```

Comment:
I don't think you need this comment.  ;)

Reply:
Removed it